### PR TITLE
Refactored selenium tests for improved readability

### PR DESCRIPTION
**Pull request by mouse1203**
Changes made to improve readability, simplification by removing unnecessary abstraction
- added table-as-dict function instead of few functions for working with the tokens table, 
- replaced static values from locators.py by locator itself in test_browser, 
- simplified menu-bar case

### CODE REVIEW

**Comment by consideRatio** [comment]
Hi @mouse1203! I didn't understand what this PR was about from the title, so I couldn't decide if it was relevant for me to contribute with a review or similar. But what PR title would been suitable then?

Looking at the files changed I think that only selenium tests are changed. Looking at the current PR description and at the changes, it seems you are not adding/removing test coverage, but that you are refactoring the tests to be easier to maintain/understand going onwards. This tells us both what code changed and for what purpose!

What do you think about a PR title like "Refactored selenium tests for improved readability"?

Another bonus would be to lead the PR description with a sentence or two to clarify what motivated the work you do before you describe the technical changes made, as a transition from the PR title towards the technical details.

File: jupyterhub/tests/selenium/test_browser.py [hunk 1]
```diff
@@ -519,85 +510,27 @@ async def open_token_page(app, browser, user):
...
-        )
+def token_table_body_as_dict(browser, table_locator):
+    # get the token table elements (excluding of the tables header)
+    table = browser.find_element(*table_locator)
+    body = table.find_element(By.TAG_NAME, 'tbody')
+    row = body.find_elements(By.CLASS_NAME, 'token-row')
+    # return dict contains elements in row and column
+    return [
+        [element.text for element in row.find_elements(By.TAG_NAME, 'td')]
+        for row in body.find_elements(By.CLASS_NAME, 'token-row')
```
Comment (minrk) [inline jupyterhub/tests/selenium/test_browser.py:521]: Should this be:

```suggestion
        for row in body.find_elements(By.TAG_NAME, 'tr')
```

The `row` above  on L517 also looks to be unused.

**Review by minrk — COMMENTED** [review]
These changes look great, thanks @mouse1203! It would be great to follow @consideRatio's feedback to add some explanation for why we are making these changes (improving readability, simplification by removing unnecessary abstraction).

@consideRatio's suggested title seems just right to me.

**Comment by mouse1203** [comment]
Hi @consideRatio , @minrk  thank you for detailed feedback!

**Comment by minrk** [comment]
Wonderful, thank you @mouse1203!

**Pull request merged**